lib/active_record/virtual_attributes.rb

```diff
@@ -141,3 +141,8 @@ def with_value_from_database(value)
 
 require "active_record/virtual_attributes/virtual_total"
 require "active_record/virtual_attributes/arel_groups"
+
+# legacy support for sql types
```

Comment:
Do we need to wrap this in some Rails version?  (Not sure what the word "legacy" refers to here)

Reply:
miq was storing this type name in our database
So legacy means miq databases

Reply:
We can clean up the message later, I think this is support for pre-extraction virtual attributes